services/app/app/components/directory-sections-review/section.js

```diff
@@ -0,0 +1,30 @@
+import Component from '@ember/component';
+import { computed } from '@ember/object';
+
+export default Component.extend({
+  classNames: ['col-md-4 col-sm-6 col-12 form-group'],
+  section: null,
+
+  isConflicting: computed('sectionIds.[]', 'selected.[]', 'section.id', function() {
+    const id = this.get('section.id');
+    if (!this.get('selected').includes(id)) return false;
+    return this.get('sectionIds').includes(id);
+  }),
+
+  disabled: computed('canSelect', function() {
+    return !this.get('canSelect');
+  }),
```

Comment:
Review everywhere `canSelect` is used -- in the original components this was based off of the current/max number selected. In fact, I'm pretty sure you can remove this everywhere on the new components -- and any properties that depend on it like this one. In this case, if `canSelect` is not passed through (likely from the controller), the control should behave as if it is disabled.